PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -144,5 +256,27 @@ def check_neighbors(step):
                     containsNeighbor[1] = True  
     
     return containsNeighbor
+ 
+ 
      
-    
\ No newline at end of file
+def issue_milestone(api):    
+    milestone_txn1 = \
+        ProposedTransaction(
+                address = Address('999999999999999999999999999999999999999999999999999999999999999999999999999999999'),
+                tag = 'BLOWBALL9TESTING9COO',
+                value = 0
+                )
+        
+    milestone_txn2 = \
+            ProposedTransaction(
+                address = Address('TESTNET9TESTING9COO9TESTNET9TESTING9COO9TESTNET9TESTING9COO9TESTNET9TESTING9COO99'),
+                tag = 'BLOWBALL9TESTING9COO',
+                value = 0
+                )
+        
+    logger.info('Sending Milestone')
+    api.send_transfer(depth=3,transfers=[milestone_txn1,milestone_txn2])
```

Comment:
I suppose this doesn't really work, as we talked about.

Reply:
It does not that's correct, I have yet to try the solution you proposed, but it is my intention to address that this week.